.circleci/images/Dockerfile

```diff
@@ -0,0 +1,100 @@
+ARG BASE_IMAGE=ubuntu:18.04
+ARG ENV_TYPE=python
+FROM ${BASE_IMAGE} AS compile-image
+
+ENV PYTHONUNBUFFERED TRUE
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install --no-install-recommends -y \
+        build-essential \
+        ca-certificates\
+        curl \
+        git \
+        openjdk-11-jdk \
+    && curl -sL https://deb.nodesource.com/setup_14.x | bash - \
+    && apt-get update \
+    && apt-get install --no-install-recommends -y \
+        nodejs
+
+FROM compile-image as conda38-image
```

Comment:
Can you document the build steps for this Docker file

Reply:
Added comments in the docker file and also updated Readme for image creation